pygeo/geo_utils/dcel.py

```diff
@@ -266,8 +266,8 @@ def buildDcel(self):
 
         for e in self.el:
 
-            h1 = DCELHedge(self.vertices[e.con[0]], self.vertices[e.con[1]], e.X, e.PID, e.uv, e.tag)
-            h2 = DCELHedge(self.vertices[e.con[1]], self.vertices[e.con[0]], e.X, e.PID, e.uv, e.tag)
+            h1 = DCELHedge(self.vertices[e.con[0]], self.vertices[e.con[1]], e.PID, e.uv, e.tag)
```

Comment:
This is a straight bug, right? Unused part of the code?

Reply:
Yeah this is just function signature mismatch, clearly untested code.